common/db/db.go

```diff
@@ -570,3 +594,12 @@ func IsMMAPV1(database *mongo.Database, collectionName string) (bool, error) {
 	_, ok := collStats[numExtents]
 	return ok, nil
 }
+
+// GetTimeseriesColNameFromBucket returns a timeseries collection name from its bucket collection name.
+func GetTimeseriesColNameFromBucket(bucketColName string) (string, error) {
```

Comment:
nit: We usually abbreviate "collection" to "coll", not "col".

Reply:
Did you change this locally but not push it?

Reply:
oops, I changed the parameter name but omitted the function name.

Reply:
Also the variable _in_ the func. Just search for "colname".